object/token_cas.go

```diff
@@ -225,6 +235,11 @@ func GenerateCasToken(userId string, service string) (string, error) {
 		}
 
 		if value != "" {
+			if k == "avatar" {
```

Comment:
all value will be string, maybe escape all value could be a good idea

Reply:
It's ok to escape all value. if other values have special characters such as '&', they can also make crash.